axelor-supplychain/src/main/resources/views/SaleOrderLine.xml

```diff
@@ -48,58 +163,86 @@
     </panel>
   </form>
 
+  <!-- ACTION VIEW -->
+
+  <action-view name="action-view-sale-order-line-allocated-qty-wizard"
+    title="Change allocated qty" model="com.axelor.apps.sale.db.SaleOrderLine">
+    <view type="form" name="allocated-qty-sale-order-line-form"/>
+    <view-param name="popup" value="reload"/>
+    <view-param name="show-toolbar" value="false"/>
+    <view-param name="show-confirm" value="false"/>
+    <view-param name="popup-save" value="false"/>
+    <view-param name="forceEdit" value="true"/>
+    <context name="_showRecord" expr="eval: id"/>
+  </action-view>
+
+  <action-view name="action-view-sale-order-line-requested-reserved-qty-wizard"
+    title="Change requested reserved qty" model="com.axelor.apps.sale.db.SaleOrderLine">
+    <view type="form" name="requested-reserved-qty-sale-order-line-form"/>
+    <view-param name="popup" value="reload"/>
+    <view-param name="show-toolbar" value="false"/>
+    <view-param name="show-confirm" value="false"/>
+    <view-param name="popup-save" value="false"/>
+    <view-param name="forceEdit" value="true"/>
+    <context name="_showRecord" expr="eval: id"/>
+  </action-view>
+
+  <!-- ACTION GROUP -->
+
   <action-group name="action-supplychain-group-sale-saleorderline-onnew">
     <action name="action-group-sale-saleorderline-onnew"/>
     <action name="action-record-sale-order-line-fill-request-qty"/>
     <action name="action-sale-order-line-group-manage-attrs"/>
+    <action name="action-sale-order-line-group-manage-analytic"/>
   </action-group>
 
+  <action-group name="action-sale-order-line-group-manage-attrs">
+    <action name="action-sale-order-line-attrs-hide-panel"/>
+    <action name="action-sale-order-line-attrs-fill-ctx"/>
+  </action-group>
 
-  <action-method name="action-method-sale-order-line-available-and-allocated-stock"
-    model="com.axelor.apps.sale.db.SaleOrderLine">
-    <call class="com.axelor.apps.supplychain.web.SaleOrderLineController"
-      method="fillAvailableAndAllocatedStock"/>
-  </action-method>
-
-  <action-method name="action-method-sale-order-line-change-reserved-qty">
-    <call class="com.axelor.apps.supplychain.web.SaleOrderLineController"
-      method="changeReservedQty"/>
-  </action-method>
-
-  <action-method name="action-supplychain-sale-order-line-allocate-all">
-    <call class="com.axelor.apps.supplychain.web.SaleOrderLineController" method="allocateAll"/>
-  </action-method>
+  <action-group name="action-supplychain-group-sale-saleorderline-onload">
```

Comment:
Please rename to `supplychain-action-sale-order-line-group-onload` to match naming conventions